README.md

```diff
@@ -0,0 +1,6 @@
+### DAKS - Azure Devito at scale using k8s and dask
+For seismic imaging, the repository shows how you can leverage open-source PDE solver [devito](https://www.devitoproject.org/) ), to perform Full-Waveform Inversion (FWI) at scale on Azure using Dask and k8s  
+a.	CPU cores  
+b.	ICL expects current code to work on “unlimited” # of cores  
+Multi GPU cores per 1 node coming -in not earlier than in a few months   
```

Comment:
Bu "multi GPU cores" what you exactly mean ? do you mean multiple GPUs attached to the same node ?

Reply:
Yes, I meant to differentiate between pure multi-GPU processing (e.g. like using 2 GPU units on an Azure NC12 vm) vs GPU powered computation distributed across multiple nodes/vms.